airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableRow.tsx

```diff
@@ -78,10 +78,9 @@ export const CatalogTreeTableRow: React.FC<StreamHeaderProps> = ({
         <Switch
           size="sm"
           checked={stream.config?.selected}
-          onChange={onSelectStream}
           onClick={stopPropagationClickHandler}
           disabled={disabled}
-          data-testid="selected-switch"
+          testId="sync"
```

Comment:
I think we were converging on `data-testid`?

Reply:
Fixed